ci/ci-test.sh

```diff
@@ -50,12 +60,12 @@ function dumpCSIControllerLogs() {
   printf "\n\n"
 }
 
-function dumpMayaAPIServerLogs() {
-  LC=$1
-  MAPIPOD=$(kubectl get pods -o jsonpath='{.items[?(@.spec.containers[0].name=="maya-apiserver")].metadata.name}' -n openebs)
-  kubectl logs --tail=${LC} $MAPIPOD -n openebs
-  printf "\n\n"
-}
+#function dumpMayaAPIServerLogs() {
+#  LC=$1
+#  MAPIPOD=$(kubectl get pods -o jsonpath='{.items[?(@.spec.containers[0].name=="maya-apiserver")].metadata.name}' -n openebs)
+#  kubectl logs --tail=${LC} $MAPIPOD -n openebs
+#  printf "\n\n"
+#}
```

Comment:
Mayaapiserver logs are not required anymore?

Reply:
NO